sale_quotation_capable_vendor_rfq/demo/vendors.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo noupdate="0">
+    <record model="res.partner" id="partner_1">
```

Comment:
Please use readable ids. Because it's much easies to read the code later
`id="partner_doge"`
Same for all other records